Dockerfile.sdk

```diff
@@ -204,10 +205,14 @@ COPY qa/images/mug.jpg images/mug.jpg
 # Install the dependencies needed to run the client examples. These
 # are not needed for building but including them allows this image to
 # be used to run the client examples.
-RUN pip3 install --upgrade numpy pillow attrdict && \
-    find install/python/ -maxdepth 1 -type f -name \
-         "tritonclient-*linux*.whl" | xargs printf -- '%s[all]' | \
-    xargs pip3 install --upgrade
+# Skip Ubuntu 18.04 because one dependency (protobuf >= 3.20) requires
+# Python >= 3.7 which the OS default doesn't satisfy
+RUN if [ "$CMAKE_UBUNTU_VERSION" != "18.04" ]; then \
```

Comment:
Do we expect version 16.04 to be used at all? My guess is no.

Reply:
Revert this change because protobuf version is reverted to 3.19.4 which is available in 18.04